src/ProjectManager.cs

```diff
@@ -162,20 +162,48 @@ public string GetPtusaProjectsPath()
                 PInvoke.IniFile iniFile = new PInvoke.IniFile(path);
 
                 // Считывание и возврат пути каталога проектов
-                string projectsFolder = iniFile.ReadString("path", "folder_path", "");
-                if (projectsFolder.Last() == '\\')
+                string projectsFolders =
+                    iniFile.ReadString("path", "folder_path", "");
+                string[] projectsFolderArray = projectsFolders.Split(';');
+                string projectsFolder = "";
+                bool firstPathIsSaved = false;
+                string firstPath = "";
+                foreach (string pathFromArray in projectsFolderArray)
                 {
-                    return projectsFolder;
+                    if (pathFromArray != "")
+                    {
+                        if (firstPathIsSaved == false)
+                        {
+                            firstPath = pathFromArray;
+                            firstPathIsSaved = true;
+                        }
+                        projectsFolder = pathFromArray;
+                        if (projectsFolder.Last() != '\\')
+                        {
+                            projectsFolder += '\\';
+                        }
+                        string projectsPath = projectsFolder + projectName;
+                        if (Directory.Exists(projectsPath))
+                        {
+                            return projectsFolder;
+                        }
+                    }
+                }
+
+                if (firstPathIsSaved == false && firstPath != "")
```

Comment:
```suggestion
                if (firstPathIsSaved == false && firstPath == "")
```

Ошибка в логике. Если путь не сохраняли и он пуст (а не наоборот), тогда ошибка.